hazelcast-jet-core/src/main/java/com/hazelcast/jet/aggregate/CoAggregateOperationBuilder.java

```diff
@@ -39,17 +39,9 @@
 
 /**
  * Offers a step-by-step API to create an aggregate operation that
- * accepts multiple inputs. You must supply this kind of operation to a
- * co-aggregating pipeline stage. Most typically you'll need this builder
- * if you're using the {@link StageWithWindow#aggregateBuilder()}. For
- * two-way or three-way co-aggregation you can use {@link
- * AggregateOperations#aggregateOperation2} and {@link AggregateOperations#aggregateOperation3}.
- * <p>
- * To obtain this builder, call {@link AggregateOperations#coAggregateOperationBuilder()}.
- * This builder is suitable only if you'll use independent aggregate
- * operations on each input and expect to receive a separate result for
- * each of them. You can combine the individual results in the final
- * step by providing a {@code finishFn}.
+ * accepts multiple inputs. To obtain it, call {@link
+ * AggregateOperations#coAggregateOperationBuilder()}. and refer to that
+ * method's Javadoc for further details.
  */
```

Comment:
I'd say only
```
/** See {@link AggregateOperations#coAggregateOperationBuilder()} */
```
in all three cases in this PR. Much quicker to read and you have to go there anyway. No need to repeat the first sentence. Same as [here](https://github.com/hazelcast/hazelcast-jet/blob/master/hazelcast-jet-core/src/main/java/com/hazelcast/jet/pipeline/SinkBuilder.java#L35).

Reply:
I prefer it this way because it tells you something about the class when you're browsing around. So you know whether you want to follow another link or you're not interested. In any case, these changes just align the descriptions with everything else.